### PR TITLE
complete merge sorted lists

### PR DESCRIPTION
**Pull request by yinggoings**

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
+
+    return sorted(merged)
```
Comment (roshni-patel) [inline functions/merge_sorted_lists.py:4]: I like this short and simple solution, Ying. How can we do this without using the sorted() helper function?

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
```
Comment (roshni-patel) [inline tests/merge_sorted_lists_test.py:3]: Should we add a test to check for merging with empty lists?

**Review by roshni-patel — APPROVED** [review]
Looks good to me, Ying!
